### PR TITLE
Bump to 1.0

### PR DESCRIPTION
**Pull request by levb**
As agreed in the team meeting, bumping to 1.0

### CODE REVIEW

**Review by hanzei — APPROVED** [review]
:tada:

**Review by aaronrothschild — APPROVED** [review]

**Comment by DHaussermann** [comment]
@levb 0/5 - should the min version be `v6.6.0` on this?

I believe we have before set min server version to a release that does not yet exist. This should not affect servers with cloud licenses. 

I'm asking because the webapp changes to support will be in `v6.6.0` so no bindings will be available to `v6.5.0` severs.

**Comment by hanzei** [comment]
Good idea @DHaussermann  :+1:

**Comment by mickmister** [comment]
@DHaussermann `min_server_version` is now set to `6.6.0`

**Comment by DHaussermann** [comment]
I deployed and tested with this. The version bump worked as expected and there were no obvious issues with the teat-app. However, Some issues were found with Service Now. Service Now happy-path had been tested relatively recently (I believe within the last 2 weeks) but I don't recall seeing these. 

https://github.com/mattermost/mattermost-app-servicenow/issues/41
https://github.com/mattermost/mattermost-app-servicenow/issues/42
https://github.com/mattermost/mattermost-app-servicenow/issues/43

Can we triage these issues and discuss possible causes before we cut the `v1.0.0` release?   

cc @levb, @mickmister and @catalintomai

**Comment by levb** [comment]
I am trying to get the `test-e2e` pass, unfortunately it crashes on me locally (for an obscure reason that does not appear to be common), so I'll have to experiment here.

**Comment by levb** [comment]
Had to reset min_server to 6.4 to pass the E2E test.

@mickmister @hanzei @DHaussermann We can't have the v6.6 requirement until the cloud/community are on 6.6. Should we ship with min_server 6.4 for now, even if not true, and update shortly after?

**Review by catalintomai — APPROVED** [review]

File: plugin.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
     "description": "This plugin powers the Mattermost Apps Framework and is required for any Apps to run",
     "homepage_url": "https://mattermost.com/marketplace/app-framework",
     "support_url": "https://github.com/mattermost/mattermost-plugin-apps/issues",
-    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-apps/releases/tag/v0.7.0",
-    "version": "0.7.0",
-    "min_server_version": "6.2.0",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-apps/releases/tag/v1.0.0",
+    "version": "1.0.0",
+    "min_server_version": "6.4.0",
```
Comment (mickmister) [inline plugin.json:9]:
```suggestion
    "min_server_version": "6.6.0",
```

**Comment by mickmister** [comment]
Both the `e2e-cypress-tests-pinned ` and `test-e2e-postgres11` jpbs have this log in their job output:

```json
{"level":"error","ts":1649196738.8115306,"caller":"mlog/log.go:251","msg":"Unable to activate plugin","plugin_id":"com.mattermost.apps","error":"plugin requires Mattermost 6.6.0: com.mattermost.apps"}
```

though `e2e-cypress-tests-pinned` passes and `test-e2e-postgres11` does not. I tried updating the server version in the circleci job to use the commit at https://github.com/mattermost/mattermost-server/commit/2aa06529c50a56710932b4a1c5fb7bbc11e9418b, which bumps the server's minor version to `6.6.0`. I'm not sure what else is needed to pass the `min_server_version` check.

**Comment by DHaussermann** [comment]
Tested and passed
- Smoke tested the `v1.0.0`:
  - Happy path installed Service Now app
  - Config can save
  - User can oAuth connect
  - User can create ticket is Service Now
  - Input box bindings, Channel header bindings and Post menu bindings are working
  - Spot checked enable/disable app and other functions
- Version bump correct at `v1.0.0`
- Confirmed min server version now set to `6.6.0`
- Some issues are still being investigated but - No known issues are holding up this `v1.0.0` release

LGTM!

**Comment by hanzei** [comment]
@mickmister I think you need to update the commit in `go.mod`

**Comment by mickmister** [comment]
@catalintomai @DHaussermann @hanzei CI is passing now :tada:

Is this good to merge?

**Review by hanzei — COMMENTED** [review]
LGTM

**Pull request merged**